app/models/order.rb

```diff
@@ -7,7 +7,7 @@ class Order <ApplicationRecord
   has_many :items, through: :item_orders
 
   def self.by_status
-    order(status: :asc)
+    order(:status, id: :asc)
```

Comment:
hopefully this enforces order in a 100% predictable way (no more spontaneous rogue tests pls)